Makefile

```diff
@@ -172,6 +180,8 @@ release: fmtcheck embed_bundle build_docs_pdf
 
 	pushd $(RELEASE_DIR) && sha256sum * > sha256sum.txt && popd
 
+	gen_release_info
```

Comment:
@anjannath this will not work since it is a target not the command.
```
$ BUNDLE_DIR=/tmp/bundle make release
rm -rf /home/prkumar/work/github/crc/docs/build
rm -rf out
rm -f /home/prkumar/work/go/bin/crc
rm -rf release
GOARCH=amd64 GOOS=darwin go build -ldflags="-X github.com/code-ready/crc/pkg/crc/version.crcVersion=1.0.0-dev -X github.com/code-ready/crc/pkg/crc/version.bundleVersion=4.1.14 -X github.com/code-ready/crc/pkg/crc/version.commitSha=7bc3124 -extldflags='-static' -s -w -X github.com/code-ready/crc/pkg/crc/constants.bundleEmbedded=true" -o out/macos-amd64/crc ./cmd/crc
GOOS=linux GOARCH=amd64 go build -ldflags="-X github.com/code-ready/crc/pkg/crc/version.crcVersion=1.0.0-dev -X github.com/code-ready/crc/pkg/crc/version.bundleVersion=4.1.14 -X github.com/code-ready/crc/pkg/crc/version.commitSha=7bc3124 -extldflags='-static' -s -w -X github.com/code-ready/crc/pkg/crc/constants.bundleEmbedded=true" -o out/linux-amd64/crc ./cmd/crc
GOARCH=amd64 GOOS=windows go build -ldflags="-X github.com/code-ready/crc/pkg/crc/version.crcVersion=1.0.0-dev -X github.com/code-ready/crc/pkg/crc/version.bundleVersion=4.1.14 -X github.com/code-ready/crc/pkg/crc/version.commitSha=7bc3124 -extldflags='-static' -s -w -X github.com/code-ready/crc/pkg/crc/constants.bundleEmbedded=true" -o out/windows-amd64/crc.exe ./cmd/crc
/home/prkumar/work/go/bin/binappend-cli write out/linux-amd64/crc /tmp/bundle/crc_libvirt_4.1.14.crcbundle
/home/prkumar/work/go/bin/binappend-cli write out/macos-amd64/crc /tmp/bundle/crc_hyperkit_4.1.14.crcbundle
/home/prkumar/work/go/bin/binappend-cli write out/windows-amd64/crc.exe /tmp/bundle/crc_hyperv_4.1.14.crcbundle
podman run -v /home/prkumar/work/github/crc/docs:/docs:Z --rm quay.io/crcont/asciidoctor-pdf:latest -D /docs/build -o doc.pdf /docs/source/getting-started/master.adoc
mkdir release
tar cJSf release/crc-macos-amd64.tar.xz -C out crc-macos-1.0.0-dev-amd64
tar cJSf release/crc-linux-amd64.tar.xz -C out crc-linux-1.0.0-dev-amd64
cd out && zip -r /home/prkumar/work/github/crc/release/crc-windows-amd64.zip crc-windows-1.0.0-dev-amd64
  adding: crc-windows-1.0.0-dev-amd64/ (stored 0%)
  adding: crc-windows-1.0.0-dev-amd64/crc.exe (deflated 61%)
  adding: crc-windows-1.0.0-dev-amd64/LICENSE (deflated 65%)
  adding: crc-windows-1.0.0-dev-amd64/doc.pdf (deflated 68%)
pushd release && sha256sum * > sha256sum.txt && popd
~/work/github/crc/release ~/work/github/crc
~/work/github/crc
gen_release_info
make: gen_release_info: Command not found
make: *** [Makefile:180: release] Error 127
```

This should be like following.
```patch
$ git diff
diff --git a/Makefile b/Makefile
index 9904371..b8cb516 100644
--- a/Makefile
+++ b/Makefile
@@ -38,7 +38,7 @@ REPOPATH ?= $(ORG)/crc
 PACKAGES := go list ./... | grep -v /out
 SOURCES := $(shell git ls-files  *.go ":^vendor")
 
-RELEASE_INFO := $(RELEASE_DIR)/release-info.json
+RELEASE_INFO := release-info.json
 
 # Check that given variables are set and all have non-empty values,
 # die with an error otherwise.
@@ -163,7 +163,7 @@ gen_release_info:
        @sed -i s/@OPENSHIFT_VERSION@/\"$(BUNDLE_VERSION)\"/ $(RELEASE_INFO)
 
 .PHONY: release
-release: fmtcheck embed_bundle build_docs_pdf
+release: fmtcheck embed_bundle build_docs_pdf gen_release_info
        mkdir $(RELEASE_DIR)
        
        @mkdir -p $(BUILD_DIR)/crc-macos-$(CRC_VERSION)-amd64
@@ -180,7 +180,7 @@ release: fmtcheck embed_bundle build_docs_pdf
 
        pushd $(RELEASE_DIR) && sha256sum * > sha256sum.txt && popd
 
-       gen_release_info
+       @mv $(RELEASE_INFO) $(RELEASE_DIR)/$(RELEASE_INFO)
 
 HYPERKIT_BUNDLENAME = $(BUNDLE_DIR)/crc_hyperkit_$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)
 HYPERV_BUNDLENAME = $(BUNDLE_DIR)/crc_hyperv_$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)
```